sdk/communication/AzureCommunicationChat/Source/Registrar/RegistrarSettings.swift

```diff
@@ -45,17 +45,17 @@ internal enum RegistrarSettings {
     static let context = ""
 }
 
-internal enum RegistrarHeader {
+internal enum RegistrarHeader: String {
```

Comment:
Just curious why you are deciding to give the following enums a raw value?

Reply:
Hi Josh, this change is made based on Travis's suggestion on our last PR. (https://github.com/Azure/azure-sdk-for-ios/pull/936#discussion_r696718815)